lib/components/conversation/conversation_item.dart

```diff
@@ -0,0 +1,168 @@
+import 'dart:html';
+import 'dart:async';
+import 'package:katikati_ui_lib/components/logger.dart';
+
+var logger = Logger('ConversationItemView');
+
+enum ConversationItemStatus { normal, failed, pending, draft }
+
+class ConversationItemView {
+  DivElement renderElement;
+
+  ConversationItemStatus _status;
+  String _id;
+  String _message;
+  bool _choosen;
+  bool _selected;
```

Comment:
it's a bit tricky to read what the difference between these two actions is, as I think you've found as well :) `choose` as a word is more irregular, so it doesn't lend itself to the same language manipulation as `selected` does, so I'd prefer it if we can avoid it.

how would you feel about using `selected` instead of `chosen` and `checked` instead of `selected`? Even if `checked` isn't matching conceptually what the user is trying to do, I think it describes better the implementation (via a checkbox), and so it's easier to understand what the actions on this item are.

Reply:
This makes sense, a lot more readable. Fixed this.